controllers/activemqartemis_controller.go

```diff
@@ -259,85 +307,83 @@ func validateBrokerVersion(customResource *brokerv1beta1.ActiveMQArtemis) *metav
 	return nil
 }
 
-func validateExtraMounts(customResource *brokerv1beta1.ActiveMQArtemis, client rtclient.Client, scheme *runtime.Scheme) (*metav1.Condition, error) {
+func validateExtraMounts(customResource *brokerv1beta1.ActiveMQArtemis, client rtclient.Client, scheme *runtime.Scheme) (*metav1.Condition, bool) {
 
 	instanceCounts := map[string]int{}
 	var Condition *metav1.Condition
-
+	var retry bool = true
+	var ContextMessage = ".Spec.DeploymentPlan.ExtraMounts.ConfigMaps,"
 	for _, cm := range customResource.Spec.DeploymentPlan.ExtraMounts.ConfigMaps {
 		configMap := corev1.ConfigMap{}
-		found, err := validateExtraMount(cm, customResource.Namespace, &configMap, client, scheme)
-		if err != nil {
-			return nil, err
-		}
+		found := retrieveResource(cm, customResource.Namespace, &configMap, client, scheme)
 		if !found {
 			return &metav1.Condition{
 				Type:    common.ValidConditionType,
 				Status:  metav1.ConditionFalse,
 				Reason:  common.ValidConditionMissingResourcesReason,
-				Message: fmt.Sprintf("Missing required configMap %v", cm),
-			}, nil
+				Message: fmt.Sprintf("%v missing required configMap %v", ContextMessage, cm),
```

Comment:
A log line starting with a variable part its hard to search in the sources. Quotes or square brackets could help if there is no way to rephrase it

Reply:
that arg is the context message, which is the prefix based on the caller - extra mounts secrets or config maps. In practice it won't be a variable, it will be one of the CR attributes.

Reply:
maybe I should have used %s?

Reply:
that would be more clear, it is not any var

Reply:
let me fix that, thanks

Reply:
done!